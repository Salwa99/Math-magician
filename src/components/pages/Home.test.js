import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('renders when there is welcome sentence', () => {
  const home = renderer.create(<Home />).toJSON();
  expect(home).toMatchSnapshot();
});
