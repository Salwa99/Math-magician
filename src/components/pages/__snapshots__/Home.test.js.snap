// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when there is welcome sentence 1`] = `
<div>
  <div />
  <div
    className="home_page"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      Arithmetic is the branch of mathematics that deals with numbers and their operations, such as addition, subtraction, multiplication, and division. The history of arithmetic can be traced back to ancient civilizations, where early forms of counting and calculation were used for practical purposes such as trade and measurement. The earliest known evidence of arithmetic comes from ancient Sumer and Egypt. The ancient Greeks, Chinese, Indian and Islamic mathematicians also contributed to the development of arithmetic. In the 15th century, the Italian mathematician Fibonacci introduced the Hindu-Arabic numeral system to Europe, which replaced the Roman numeral system and made arithmetic calculations more efficient. Today, arithmetic is widely used in many areas of science, engineering, and everyday life.
    </p>
  </div>
</div>
`;
